examples/entity/importexport/README.txt

```diff
@@ -0,0 +1,56 @@
+####### Using MySQL as Datasource #######
+yum install mysql mysql-server mysql-libs
```

Comment:
This is OS specific and currently we support few more db instances, please mention its only specific to mysql and os details.

There are few assumptions  in the script like oozie is setup and installed... As installation directory can vary from installation to installation better get if generically  through environment variables.